Issue reported to upstream here
https://marc.info/?l=git&m=165121343215520&w=2

Hopefully will be fixed in upstream

--- git-2.48.1/config.mak.uname.orig	2025-02-05 17:16:54.439139632 +0100
+++ git-2.48.1/config.mak.uname	2025-02-05 17:20:52.462137030 +0100
@@ -188,7 +188,7 @@
 	NEEDS_SOCKET = YesPlease
 	NEEDS_NSL = YesPlease
 	SHELL_PATH = /bin/bash
-	SANE_TOOL_PATH = /usr/xpg6/bin:/usr/xpg4/bin
+	SANE_TOOL_PATH = /usr/gnu/bin
 	HAVE_ALLOCA_H = YesPlease
 	NO_STRCASESTR = YesPlease
 	NO_MEMMEM = YesPlease
